openapi_python_client/templates/property_templates/dict_property.pyi

```diff
@@ -12,6 +12,9 @@ if {{ source }} is not None:
 {% if property.required %}
 {{ destination }} = {{ source }}
 {% else %}
-{{ destination }} = {{ source }} if {{ source }} else None
+if {{ source }} is UNSET:
```

Comment:
For my context, where is the macro `transform` actually called?

Reply:
It's called frequently in the various `.pyi` templates - you'll find it easily if you search and make sure you aren't filtering to only `.py` files.